types/lodash/common/common.d.ts

```diff
@@ -6,6 +6,10 @@ declare module "../index" {
     type Omit<T, K extends keyof any> = Pick<T, Exclude<keyof T, K>>;
     type PartialObject<T> = GlobalPartial<T>;
     type Many<T> = T | readonly T[];
+    type NullOrUndefinedOnly<T> =
+        [T] extends [null] ? ([null] extends [T] ? true : false) :
```

Comment:
Out of curiosity, why are the square brackets needed? Does it work without them?